### PR TITLE
Update from template

### PR DESCRIPTION
**Pull request by ubruhin**
**Automatically created pull request to update some files to their latest version from [librepcb-libraries-admin](https://github.com/LibrePCB/librepcb-libraries-admin).**

### CODE REVIEW

**Comment by ubruhin** [comment]
Haha nice, [librepcb-libraries-admin](https://github.com/LibrePCB/librepcb-libraries-admin) seems to work :grin: 

I did not deploy to the other libraries yet because I'd like to get some feedback for the added file first. Once this is merged (and checked if the PR template works as expected), I'll deploy this change to all other libraries too.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+name:
+about:
+
+---
+
+##### SUMMARY
+<!-- Describe the changes you have made. -->
+
+
+##### PENDING POINTS / UNRESOLVED ISSUES
```
Comment (dbrgn) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Maybe this would be better?

```suggestion
##### OPEN QUESTIONS / UNRESOLVED ISSUES
```

Comment (ubruhin) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Ok, done :+1:

**Comment by ubruhin** [comment]
I fixed some issues I found by testing the PR template in a private repo, now I think it's ready.

**Pull request merged**